cura/Layer.py

```diff
@@ -62,38 +69,48 @@ def createJumps(self):
 
     def createMeshOrJumps(self, make_mesh):
         builder = MeshBuilder()
-
+        index_pattern = numpy.array([[0, 3, 2, 0, 1, 3]], dtype = numpy.int32 )
```

Comment:
You could probably make this a class member and save recreating the index_pattern every time this method is called.